docs/3.0.0-beta.x/guides/registering-a-field-in-admin.md

```diff
@@ -0,0 +1,211 @@
+# Creating a new Field in the administration panel
+
+In this guide we will see how you can create a new Field for your administration panel.
+
+## Introduction
+
+For this example, we will see how to change the WYSIWYG with [CKEditor](https://ckeditor.com/ckeditor-5/) in the **`Content Manager`** plugin by creating a new plugin which will add a new **Field** in your application.
+
+## Setup
+
+1. Create a new project:
+
+```bash
+yarn create strapi-app my-app
+```
+
+2. Generate a plugin:
+
+```bash
+strapi generate:plugin wysiwyg
+```
+
+3. Install the needed dependencies:
+
+```bash
+cd my-app/plugins/wysiwyg
+yarn add @ckeditor/ckeditor5-react @ckeditor/ckeditor5-build-classic
+```
+
+4. Start your application with the front-end development mode:
+
+```bash
+cd my-app
+yarn develop --watch-admin
+```
+
+Once this step is over all we need to do is to create our new WYSIWYG which will replace the current one the **Content Manager** plugin.
+
+### Creating the WYSIWYG
+
+In this part we will create two components:
+
+- Wysiwyg which will wrap the CKEditor with a label and the errors
+- CKEditor which will be the implementation of the new WYSIWYG
+
+#### Creating the WYSIWYG Wrapper
+
+**Path —** `./plugins/wysiwyg/admin/src/components/Wysiwyg/index.js`
+
+```js
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import { Label, InputDescription, InputErrors } from 'strapi-helper-plugin';
+import Editor from '../CKEditor';
+
+const Wysiwyg = ({
+  inputDescription,
+  errors,
+  label,
+  name,
+  noErrorsDescription,
+  onChange,
+  value,
+}) => {
+  let spacer = !isEmpty(inputDescription) ? <div style={{ height: '.4rem' }} /> : <div />;
+
+  if (!noErrorsDescription && !isEmpty(errors)) {
+    spacer = <div />;
+  }
+
+  return (
+    <div
+      style={{
+        marginBottom: '1.6rem',
+        fontSize: '1.3rem',
+        fontFamily: 'Lato',
+      }}
+    >
+      <Label htmlFor={name} message={label} style={{ marginBottom: 10 }} />
+      <Editor name={name} onChange={onChange} value={value} />
+      <InputDescription
+        message={inputDescription}
+        style={!isEmpty(inputDescription) ? { marginTop: '1.4rem' } : {}}
+      />
+      <InputErrors errors={(!noErrorsDescription && errors) || []} name={name} />
+      {spacer}
+    </div>
+  );
+};
+
+Wysiwyg.defaultProps = {
+  errors: [],
+  label: '',
+  noErrorsDescription: false,
+  value: '',
+};
+
+Wysiwyg.propTypes = {
+  errors: PropTypes.array,
+  inputDescription: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.func,
+    PropTypes.shape({
+      id: PropTypes.string,
+      params: PropTypes.object,
+    }),
+  ]),
```

Comment:
Is `inputDescription` required or should it have a default prop?

Reply:
I have updated the `defaultProps`